Mailer/app/api/v1/Handlers/Mailers/MailLogsHandler.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Remp\MailerModule\Api\v1\Handlers\Mailers;
+
+use Nette\Utils\DateTime;
+use ParseLogFilterConditionsTrait;
+use Nette\Utils\Arrays;
+use Remp\MailerModule\ActiveRow;
+use Remp\MailerModule\Api\Validation\InputValidatorTrait;
+use Remp\MailerModule\Repository\LogsRepository;
+use Tomaj\NetteApi\Handlers\BaseHandler;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+
+class MailLogsHandler extends BaseHandler
+{
+    private $logsRepository;
+
+    use InputValidatorTrait;
+    use ParseLogFilterConditionsTrait;
+
+    public function __construct(
+        LogsRepository $logsRepository
+    ) {
+        parent::__construct();
+        $this->logsRepository = $logsRepository;
+    }
+
+    public function handle($payload)
+    {
+        $request = file_get_contents("php://input");
+        $payload = $this->validateInput($request, 'logs.schema.json');
+
+        if ($this->hasErrorResponse()){
+            return $this->getErrorResponse();
+        }
+
+        $output = $this->logsRepository->all();
+
+        if (isset($payload['filter'])){
+            $output->whereOr($this->parseConditions($payload['filter']));
```

Comment:
Most probably I would make this regular `where`. There's a change I'd search for "emails sent between X and Y which are also dropped" instead of "emails sent between X and Y or emails that are dropped"

Reply:
Oh, I thought that in your previous feedback 
`"sent after DATE_A" or "delivered after DATE_A and before DATE_B"`
the or means actual OR in SQL :D

Reply:
no, it was to separate examples :).